lib/gcloud/bigquery/connection.rb

```diff
@@ -597,6 +598,10 @@ def source_format path, format
         nil
       end
 
+      def delimiter_field str #:nodoc:
```

Comment:
I don't think this check is needed. The docs say the first byte of the string is used. This method can be removed.